test/sqllogictest/cluster.slt

```diff
@@ -837,4 +837,38 @@ DROP CLUSTER t1
 ----
 COMPLETE 0
 
+# Test SHOW CREATE
+
+mode standard
+
+statement ok
+CREATE CLUSTER c1 (SIZE '1', REPLICATION FACTOR 2)
+
+query TT
+SHOW CREATE CLUSTER c1
+----
+c1
+CREATE CLUSTER "c1" (DISK = true, INTROSPECTION DEBUGGING = false, INTROSPECTION INTERVAL = INTERVAL '00:00:01', MANAGED = true, REPLICATION FACTOR = 2, SIZE = '1', SCHEDULE = MANUAL)
+
+statement ok
+DROP CLUSTER c1
+
+simple conn=mz_system,user=mz_system
+CREATE CLUSTER c1 SIZE = '1' FEATURES (ENABLE EAGER DELTA JOINS = TRUE);
+----
+COMPLETE 0
+
+query TT
+SHOW CREATE CLUSTER c1
+----
+c1
+CREATE CLUSTER "c1" (DISK = true, INTROSPECTION DEBUGGING = false, INTROSPECTION INTERVAL = INTERVAL '00:00:01', MANAGED = true, REPLICATION FACTOR = 1, SIZE = '1', SCHEDULE = MANUAL) FEATURES (ENABLE EAGER DELTA JOINS = true)
```

Comment:
We should test `ALTER CLUSTER` here too!

Reply:
Done